src/datetime.js

```diff
@@ -766,7 +768,9 @@ export default class DateTime {
     const normalized = normalizeObject(obj, normalizeUnitWithLocalWeeks);
     const { minDaysInFirstWeek, startOfWeek } = usesLocalWeekValues(normalized, loc);
 
-    const tsNow = Settings.now(),
+    const tsNow = isUndefined(opts.refrenceDate)
+      ? friendlyDateTime(opts.refrenceDate).toUnixInteger()
+      : opts.refrenceDate,
```

Comment:
I'm not a member of the Luxon team, but here's my thoughts:

- maybe `tsNow` would be better named `tsRef`
- I don't think `opts.refrenceDate` should be used if it's `undefined`